data/default_os_images.json

```diff
@@ -82,5 +82,17 @@
         "cpu_architecture": "arm64",
         "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-aarch64-live.aarch64.iso",
         "version": "412.86.202212081411-0"
+    },
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "ppc64le",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-ppc64le-live.ppc64le.iso",
```

Comment:
```suggestion
        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/pre-release/latest-4.13/rhcos-4.13.0-ec.3-ppc64le-live.ppc64le.iso",
```

Reply:
Bumped all 4.13 dev preview images to ec.3.

Now running skipper make generate and will push once done.

Reply:
Done ^